templates/projects/application/shadowy_super_code.md

```diff
@@ -0,0 +1,66 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+"Shadowy Super Coder"
+
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+This is a decentralized gaming utilizing IPFS for images and level storage.
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+Shadowy Super Coder is a blockchain based game that uses ChainLink for random character generation. This allows player to import data from NFT marketplaces to use in game and trade with other players. The player also has the ability to interact ingame making purchases using an in game currency that support ths ERC20 token standard.
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+After completing HACKFS the games current status is under development. The team is looking to ship a MVP in the next 2 months. 
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+
+![Screen Shot 2021-09-16 at 2 25 52 PM](https://user-images.githubusercontent.com/681817/133665265-6ada1b70-4994-433e-af08-c50d7c0e167d.png)
+
+
+## Target Audience
+<!--Describe who will be your project's users-->
+This dApp is for anyone that enjoys browser based game and interested in crypto and decentralization. The gameplay mechanics are such that the game will be easily accesible to player new to the blockchain game experiences and 
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/KBryan/typescript_game_hackfs 
+
+## Website
+<!--Link your website if available-->
+https://gateway.pinata.cloud/ipfs/QmaMA8w3w1B7zRJdDMNhoT3AHKcHtB1cH4aM6AcwoEPGji/ssc/ 
+## Docs
+<!--Including a link to your project docs!-->
+Currently in Developer
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+Kwame Bryan = Lead Developer
```

Comment:
Could you please fix the format? There of your team members are listed at the same line.